### PR TITLE
BE-4-add-migratiions

### PR DESCRIPTION
**Pull request by viktoriya-okseniuk**

### CODE REVIEW

File: src/app.service.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 import { mainInterface } from './app.main.interface';
 
 @Injectable()
 export class AppService {
-  async healthCheck(): Promise<mainInterface> {return {'status_code': 200, 'detail': 'ok', 'result': 'working'}}
+  private readonly logger = new Logger(AppService.name);
+
+  async healthCheck(): Promise<mainInterface> {
+    this.logger.log('healthCheck log')
+    return {'status_code': 200, 'detail': 'ok', 'result': 'working'}
```
Comment (Ivanus-Volodymyr) [inline src/app.service.ts]: use HttpStatus

File: src/logger/logger.service.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ConsoleLogger, Injectable, Scope } from '@nestjs/common';
+import { loggerProps } from './logger.interface';
+
+@Injectable({ scope: Scope.TRANSIENT })
+export class MyLogger extends ConsoleLogger {
+  error({message, stack, context}: loggerProps) {
+    // add your tailored logic here
+    super.error(message, stack, context);
+  }
+}
```
Comment (Ivanus-Volodymyr) [inline src/logger/logger.service.ts:29]: extend

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

File: src/app.service.ts (hunk 1 above)
Comment (viktoriya-okseniuk) [reply src/app.service.ts]: Http code works on local , but not working in Docker. need to figure it out((

Comment (Ivanus-Volodymyr) [reply src/app.service.ts]: HttpStatus.OK

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

**Pull request merged**